babel.go

```diff
@@ -23,6 +23,7 @@ func (t *babelTransformer) Done() {
 
 var once = &sync.Once{}
 var globalpool chan *babelTransformer
+var babelProg = goja.MustCompile("babel.js", string(_MustAsset("babel.js")), false)
```

Comment:
I'd prefer the compile step be done in the once function however and the error captured and retured if not nil. I also think it should be in strict mode unless that doesn't work.

i.e.

```go
// ...
var babelProg *Program

func Init(poolSize int) (err error) {
	once.Do(func() {
		babelSrc, err = _Asset("babel.js")
		if err != nil {
			return
		}
		babelProg, err = goja.Compile("babel.js", string(babelSrc), true)
		if err != nil {
			return
		}
		// ...
	})

	return err
}
// ...
```